### PR TITLE
Tests/add

### PR DESCRIPTION
**Pull request by Tzumx**
Tests (initial version)

### CODE REVIEW

File: tests/test_bookings.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+    # skips the authentication
+    def skip_auth():
+        pass
+    app.dependency_overrides[users_utils.get_current_user] = skip_auth
+
+    yield client
+
+
+def test_routers_bookings(client):
```
Comment (mikaserova) [inline tests/test_bookings.py]: what exactly are you testing here?
there are different endpoints in this test, and if it fails it's hard to understand which one is broken. Also you test only positive scenarios here.

1 unit test usually covers 1 function and for better readability its name should be test_function_name_some_conditions: test_create_booking_insufficient_permissions etc

File: tests/test_bookings.py [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    with patch('crud.bookings.set_booking_status') as mock:
+        mock.return_value = bookings_schema.BookingInfo(
+            **(booking_data | booking_data_add))
+        response = client.patch('/bookings/11/is_active?is_active=True')
+        assert response.status_code == 200
+        data = response.json()
+        assert data['id'] == booking_data_add['id']
+
+
+def test_crud_bookings(client):
```
Comment (mikaserova) [inline tests/test_bookings.py]: Split this into separate tests

File: tests/test_guests.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+}
+guest_2_data = {
+    'name': 'name_of_guest2',
+    'email': 'nam2e@guest.com',
+    'phone': '98746543210',
+}
+
+
+@pytest.fixture
+def client(client: TestClient):
```
Comment (mikaserova) [inline tests/test_guests.py]: you can use different name here to make it more clear: client_no_auth

**Review by mikaserova — COMMENTED** [review]
Test should be spliced into smaller ones

File: tests/test_guests.py (hunk 3 above)
Comment (Tzumx) [reply tests/test_guests.py]: Done

File: tests/test_bookings.py (hunk 2 above)
Comment (Tzumx) [reply tests/test_bookings.py]: Done

File: tests/test_bookings.py [hunk 4]
```diff
@@ -46,66 +55,125 @@ def skip_auth():
...
+        response = client_no_auth.post('/bookings', json=booking_data)
         assert response.status_code == 200
         data = response.json()
         assert data['id'] == booking_data_add['id']
         assert data['description'] == booking_data['description']
 
+        response = client_no_auth.post('/bookings', json=roomtype_data)
+        assert response.status_code == 422
+
+        try:
```
Comment (mikaserova) [inline tests/test_bookings.py]: use ( match is optional)
```
with purest.raises(ValidationError, match="error_message): 
```

**Review by mikaserova — APPROVED** [review]
It's better now
Just use pytest.raises instead try..except in tests
Approving in advance

File: tests/test_bookings.py (hunk 4 above)
Comment (Tzumx) [reply tests/test_bookings.py]: Done, thanks

**Pull request merged**